### PR TITLE
Fix some style issues and add CI (#56)

### PR DESCRIPTION
**Pull request by ChillerDragon**

### CODE REVIEW

File: deploy.sh
Comment (Zebiano) [inline deploy.sh:291]: Removing the `difference` variable and then attempting to use it on line `289` won't work. You'd have to change `289` to `"$now"`.

File: .github/workflow/main.yml
Comment (Zebiano) [inline .github/workflow/main.yml:18]:
```suggestion
      - name: Dependencies
```

**Review by Zebiano — CHANGES_REQUESTED** [review]
Hey! Thank you for this Pull Request. I like the CI addition, it's an interesting idea and will definitely come in handy! I'd just like to wait for PR #59 to be pushed to then implement it.

Also, I've made some comments that maybe you could address? Though honestly, I'd rather wait until #59 gets merged because the code there is quite different and maybe some or all of the changes you've made won't apply anymore. 

Thank you!

File: afk-daily.sh
Comment (ChillerDragon) [reply afk-daily.sh:1]: Woopsie

**Comment by ChillerDragon** [comment]
Yea thought so I realized that #59 is there when I already submitted my pr. Yea so let's wait.

File: deploy.sh
Comment (kevingrillet) [reply deploy.sh:59]: It's possible if the script doesn't have the right to use `mkdir`, that will raise a warning, then `cd` will do the same, and we will continue in the wrong folder :/

File: .github/workflow/main.yml
Comment (kevingrillet) [inline .github/workflow/main.yml:23]: Mmh, I think we need to remove more things!

```
    "shellcheck.exclude": [
        "1090", // https://github.com/koalaman/shellcheck/wiki/SC1090
        "1091", // https://github.com/koalaman/shellcheck/wiki/SC1091
        "2034", // https://github.com/koalaman/shellcheck/wiki/SC2034
        "2154", // https://github.com/koalaman/shellcheck/wiki/SC2154
        "3057", // https://github.com/koalaman/shellcheck/wiki/SC3057
        "3060"  // https://github.com/koalaman/shellcheck/wiki/SC3060
    ],
```

**Review by kevingrillet — CHANGES_REQUESTED** [review]

**Comment by Zebiano** [comment]
Thank you very much for this PR!

**Pull request merged**
